TealiumCrashModuleExample/TealiumCrashModuleExample/ContentView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ContentView.swift
+//  TealiumCrashModuleExample
+//
+//  Created by Enrico Zannini on 19/07/22.
+//
+
+import SwiftUI
+import TealiumSwift
+import TealiumCrashModule
+
+struct ContentView: View {
+    var body: some View {
+        Group {
+            Button {
+                tealium.track(TealiumEvent("Some Event"))
+            } label: {
+                Text("Track")
+            }.padding()
+            Button {
+                CrashReporter.invokeCrash(name: "asd", reason: "asd")
```

Comment:
there might be a better example app crash than `asd` haha

Reply:
definitely 😄